### PR TITLE
chore: remove lockfiles

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
Follow-up of https://github.com/remix-run/remix/pull/3110

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -24,6 +24,8 @@ jobs:
 
       - name: 📥 Download deps
         uses: bahmutov/npm-install@v1
+        with:
+          useLockFile: false
```
Comment (MichaelDeBoey) [inline .github/workflows/deploy.yml:28]: @kentcdodds If you want, I can remove these again in `remix.init`

**Comment by kentcdodds** [comment]
Any idea for what we should do about this? https://github.com/remix-run/indie-stack/runs/6358741070?check_suite_focus=true

**Comment by MichaelDeBoey** [comment]
@kentcdodds Probably a good idea to remove `package-lock.json` from the `Dockerfile`?
If that's not preferred, we should install first I think 🤔

**Comment by kentcdodds** [comment]
I don't think we want to remove it from the Dockerfile because users will want it in there.

**Comment by MichaelDeBoey** [comment]
Then we should do an install first.

We can't know what file people will have though, as we don't know which package manager they will use.

https://github.com/remix-run/remix/pull/3076 will already pass the used one to `remix.init` though, so if that one's merged/released, we _could_ do `${packageManager} install` & update `Dockerfile` in `remix.init` if we want to

**Comment by kentcdodds** [comment]
Yeah, let's be a little more sophisticated 👍 I just merged that PR. Thanks!

**Comment by MichaelDeBoey** [comment]
We'll need to wait for the next release to make it happen though. 😕

**Pull request merged**